### PR TITLE
32-cycle-restarter

### LINKED ISSUE #32: [FEATURE] Cycle restarter
## 🚀 Feature

When the banner of cycle counter is clicked should display the option to restart it.

### Motivation

Give flexibility to the user.

### Implementation

When clicked, it should give the option to restart the counter.

### Additional context

Image

### PR DESCRIPTION
**Pull request by JM-Pascual**
## Motivation

Closes #32 

## Summary of changes

Now the user can click over the "Completed pomodoros" sign and reset the count.

### CODE REVIEW

File: src/components/Timer/timer.js [hunk 1]
```diff
@@ -65,6 +66,17 @@ export default function Timer(props) {
     }
   }
 
+  function resetPomodoroCount() {
+    // Display an alert to confirm the reset
+    const isConfirmed = window.confirm("Are you sure you want to reset the pomodoro count?");
+
+    // If user confirms, reset the count
+    if (isConfirmed) {
+      setCycles(0);
```
Comment (levdevs) [inline src/components/Timer/timer.js:75]: You should also update the local storage here.

**Review by levdevs — CHANGES_REQUESTED** [review]
Button works as expected, however the local storage values are not retained when refreshing the page.

File: src/components/Timer/timer.js (hunk 1 above)
Comment (JM-Pascual) [reply src/components/Timer/timer.js:75]: Fixed! 👍🏼

**Review by joac001 — APPROVED** [review]
Okay

**Review by levdevs — APPROVED** [review]
Works flawlessly now, good job

**Pull request merged**